### PR TITLE
Feat: Landing page

### PR DESCRIPTION
**Pull request by nishasen**
Added landing page, it has two ways of navigation, one using product house and another using category name. Also I have added four themes which are related to the project theme.
image
image
image

### CODE REVIEW

File: diagon-alley/src/Components/House/House.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.housepane {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
```
Comment (sadathshariff) [inline diagon-alley/src/Components/House/House.css:4]: You can extract this as a utility class since you are using it in many places

Comment (nishasen) [reply diagon-alley/src/Components/House/House.css:4]: I did not make any utility for this in CL so kept it this way now.

**Review by sadathshariff — APPROVED** [review]
Good Naming convention and liked how you are using useTheme for Changing the Theme based on the Houses

**Review by aishwarya-bm — APPROVED** [review]
Great breaking and reuse of components.
The code looks clean with good usage of props and contexts.

**Pull request merged**